core/src/test/scala/unit/kafka/admin/DescribeConsumerGroupTest.scala

```diff
@@ -22,297 +22,254 @@ import java.util.concurrent.TimeUnit
 
     val cgcArgs = Array("--bootstrap-server", brokerList, "--describe", "--group", group)
     val opts = new ConsumerGroupCommandOptions(cgcArgs)
-    val consumerGroupCommand = new KafkaConsumerGroupService(opts)
+    consumerGroupService = new KafkaConsumerGroupService(opts)
 
     TestUtils.waitUntilTrue(() => {
-        val (state, assignments) = consumerGroupCommand.describeGroup()
+        val (state, assignments) = consumerGroupService.describeGroup()
         state == Some("Stable") &&
         assignments.isDefined &&
         assignments.get.count(_.group == group) == 1 &&
         assignments.get.filter(_.group == group).head.consumerId.exists(_.trim != ConsumerGroupCommand.MISSING_COLUMN_VALUE) &&
         assignments.get.filter(_.group == group).head.clientId.exists(_.trim != ConsumerGroupCommand.MISSING_COLUMN_VALUE) &&
         assignments.get.filter(_.group == group).head.host.exists(_.trim != ConsumerGroupCommand.MISSING_COLUMN_VALUE)
     }, "Expected a 'Stable' group status, rows and valid values for consumer id / client id / host columns in describe group results.")
-
-    consumerGroupCommand.close()
-    executor.shutdown()
   }
 
   @Test
   def testDescribeExistingGroupWithNoMembersWithNewConsumer() {
+    TestUtils.createOffsetsTopic(zkUtils, servers)
     // run one consumer in the group consuming from a single-partition topic
-    val executor = new ConsumerGroupExecutor(brokerList, 1, group, topic)
+    consumerGroupExecutor = new ConsumerGroupExecutor(brokerList, 1, group, topic)
 
     val cgcArgs = Array("--bootstrap-server", brokerList, "--describe", "--group", group)
     val opts = new ConsumerGroupCommandOptions(cgcArgs)
-    val consumerGroupCommand = new KafkaConsumerGroupService(opts)
+    consumerGroupService = new KafkaConsumerGroupService(opts)
 
     TestUtils.waitUntilTrue(() => {
-      val (state, _) = consumerGroupCommand.describeGroup()
+      val (state, _) = consumerGroupService.describeGroup()
       state == Some("Stable")
     }, "Expected the group to initially become stable.")
 
     // stop the consumer so the group has no active member anymore
-    executor.shutdown()
+    consumerGroupExecutor.shutdown()
 
     TestUtils.waitUntilTrue(() => {
-        val (state, assignments) = consumerGroupCommand.describeGroup()
+        val (state, assignments) = consumerGroupService.describeGroup()
         state == Some("Empty") &&
         assignments.isDefined &&
         assignments.get.count(_.group == group) == 1 &&
         assignments.get.filter(_.group == group).head.consumerId.exists(_.trim == ConsumerGroupCommand.MISSING_COLUMN_VALUE) && // the member should be gone
         assignments.get.filter(_.group == group).head.clientId.exists(_.trim == ConsumerGroupCommand.MISSING_COLUMN_VALUE) &&
         assignments.get.filter(_.group == group).head.host.exists(_.trim == ConsumerGroupCommand.MISSING_COLUMN_VALUE)
     }, "Expected no active member in describe group results.")
```

Comment:
Thanks @rajinisivaram, I can include the state and assignment in the error message, that should be more informative. With regards to the more important question of _why_ it fails, not sure. Not sure if I'll have time to investigate today, so would you be OK if I file a separate JIRA for that? `testDescribeGroupWithNewConsumerWithShortInitializationTimeout` has been failing somewhat often in Jenkins and hopefully this PR will make it more stable (we'll have to see if it solves the issue altogether though).